docs/sdk/php/usage.md

```diff
@@ -0,0 +1,238 @@
+---
+id: usage
+title: Como iniciar
+tags:
+  - api
+  - php
+---
+
+## Instalando
+
+É necessário ter o PHP com versão superior à 7.4 instalado com o [Composer](https://getcomposer.org).
+
+A maneira mais fácil de instalar o SDK é com o comando abaixo que instala as dependências necessárias com o Composer:
+
+```bash
+$ composer require open-pix/php-sdk guzzlehttp/guzzle guzzlehttp/psr7
+```
+
+Dessa forma, o SDK, um cliente HTTP (`guzzlehttp/guzzle`) e uma implementação da [PSR-17](https://www.php-fig.org/psr/psr-17/) (`guzzlehttp/psr7`) serão instalados.
+
+## Dependências
+
+O SDK depende de implementações das PSRs 17 e [18](https://www.php-fig.org/psr/psr-18/).
+
+A PSR-17 é usada para fornecer interfaces para mensagens HTTP, tanto requisições quanto respostas. Por outro lado, a PSR-18 é usada para fornecer interfaces para clientes HTTP.
+
+Sendo interfaces, é necessário ter uma implementação instalada, como `guzzlehttp/guzzle` para a PSR-18 e `guzzlehttp/psr7` para a PSR-17.
+
+As PSRs são utilizadas para não depender diretamente de clientes HTTP, deixado a cargo do desenvolvedor.
+
+## Criando o cliente
+
+O ponto de entrada do SDK é um `Client`:
+
+```php
+use OpenPix\PhpSdk\Client;
+
+// Para utilizar OpenPix
+$client = Client::create("coloque seu appid aqui");
+```
+
+O método `create` cria um novo cliente a partir de um ID de aplicativo obtido no [site da OpenPix](https://app.wooopenpix.com.br/home/applications/tab/list).
```

Comment:
fix it link
[app.openpix.com.br/](https://app.openpix.com.br/home/applications/tab/list)